azurerm/internal/services/policy/tests/resource_arm_policy_set_definition_test.go

```diff
@@ -179,13 +177,12 @@ POLICY_RULE
     }
   }
 PARAMETERS
-
 }
 
 resource "azurerm_policy_set_definition" "test" {
   name         = "acctestpolset-%d"
   policy_type  = "Custom"
-  display_name = "acctestpolset-%d"
+  display_name = "acctestpolset-display-%d"
```

Comment:
Could we use some uppercase letters here?

Reply:
Done